admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -56,7 +56,23 @@ ensure_helper_file_present() {
         fi
     fi
 
-    echo "$DIR/$NAME"
+    readlink -e "$DIR/$NAME"
```

Comment:
Nit: why is this needed? Gonna fail on BSD readlinks. Not a huge deal, but if not strictly needed, could do without :)

Reply:
It's needed because the function sometimes returns relative paths without it and docker -v fails if it's being passed a relative path, it only works with absolute paths.

Reply:
👍 thanks for the explanation

I think `realpath` is more portable than `readlink -e` though

Reply:
If you want, I can change it to realpath, I've got no particular preference wrt this.

Reply:
If you don't mind, that'd be great. So people can still dev on this on Mac. Thanks!